src/main/java/core/basesyntax/store/Storage.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.store;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class Storage {
+    private static final Map<String, Integer> fruits = new ConcurrentHashMap<>();
```

Comment:
Why concurrent map? do you use multithreading?

Reply:
![Screenshot_162_](https://github.com/user-attachments/assets/39ced77f-4329-415b-a2a9-0464b17c3ba2)

This was a wish during the review of the previous task from ai mentor)
Fix back?